### PR TITLE
SQHELM-118 fix unsupported wget parameter

### PR DESCRIPTION
**Pull request by jCOTINEAU**

### CODE REVIEW

File: charts/sonarqube-dce/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [9.3.0]
```
Comment (carminevassallo) [inline charts/sonarqube-dce/CHANGELOG.md:4]: We should align the changelog with the new chart version :)

File: charts/sonarqube-dce/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sonarqube-dce
 description: SonarQube offers Code Quality and Code Security analysis for up to 27 languages. Find Bugs, Vulnerabilities, Security Hotspots and Code Smells throughout your workflow.
 type: application
-version: 8.2.2
+version: 8.3.0
```
Comment (carminevassallo) [inline charts/sonarqube-dce/Chart.yaml:5]: We should bump the patch version for bug fixes..

File: charts/sonarqube-dce/Chart.yaml [hunk 3]
```diff
@@ -24,6 +24,8 @@ maintainers:
     email: jeremy.cotineau@sonarsource.com
 annotations:
   artifacthub.io/changes: |
+    - kind: fixed
+      description: "Fixed unsupported wget parameter `--proxy off` with `--no-proxy`"
```
Comment (carminevassallo) [inline charts/sonarqube-dce/Chart.yaml:28]: are these descriptions rendered as markdown by the changelog section of ArtifactHub.io?

File: charts/sonarqube/Chart.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sonarqube
 description: SonarQube offers Code Quality and Code Security analysis for up to 27 languages. Find Bugs, Vulnerabilities, Security Hotspots and Code Smells throughout your workflow.
 type: application
-version: 9.4.1
+version: 9.5.0
```
Comment (carminevassallo) [inline charts/sonarqube/Chart.yaml:5]: same comment about the patch version..

**Review by carminevassallo — CHANGES_REQUESTED** [review]

**Review by carminevassallo — APPROVED** [review]
LGTM!

**Comment by bbodenmiller** [comment]
Any chance this can be backported to 8.x chart? Or a workaround that folks affected by this could put in place while waiting? We manually removed readiness probe but it seems there should be a better solution.

**Comment by jCOTINEAU** [comment]
Hello @bbodenmiller, it is not planed yet, i will grab some information and get back to you soon.

Right now as a quick workaround you could use helm post render to kustomize those probe and replace them with the correct options.

I am really sorry for the inconvenience.

**Pull request merged**